### ISSUE TITLE
Add activation function to Metrics 

### ISSUE THREAD

**Issue by radka-j** [alan-turing-institute/ARC-MTQE#46]
Pass model output through softmax or sigmoid based on the loss function (this is used in the validation step and and when making predictions).

Also update docstrings to indicate that the outputs are logits (?) rather than probabilities.
